keyboards/planck/keymaps/navi/config.h

```diff
@@ -27,4 +34,27 @@
 /* override number of MIDI tone keycodes (each octave adds 12 keycodes and allocates 12 bytes) */
 //#define MIDI_TONE_KEYCODE_OCTAVES 2
 
+// Most tactile encoders have detents every 4 stages
+#define ENCODER_RESOLUTION 4
+
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
`#define PREVENT_STUCK_MODIFIERS` is the default in QMK since #3107. You can delete this line.

```suggestion
```